pysumma/Simulation.py

```diff
@@ -30,12 +30,19 @@ def __init__(self, filepath):
         self.output_prefix = FileManagerOption('output_prefix', self.filepath)
         self.decision_obj = Decisions(self.setting_path.value + self.decision_path.value)
 
-    def execute(self):
+    def execute(self,     run_suffix):
+        self.run_suffix = run_suffix
         if self.executable == '':
             raise ValueError('No executable defined. Set as "executable" attribute of Simulation')
         else:
             cmd = "{} -p never -s {}       -m {}".format(self.executable, self.run_suffix, self.filepath)
             subprocess.run(cmd, shell=True)
+            out_file_path = 	self.output_path.filepath + \
```

Comment:
This is somewhat brittle, but maybe beyond the scope of this pull request.  We should figure out a better way to pick up where output comes from.  I'll open an issue on this topic.  I think it's fine here for now, though.